sklearn/linear_model/passive_aggressive.py

```diff
@@ -81,20 +89,24 @@ class PassiveAggressiveClassifier(BaseSGDClassifier):
     K. Crammer, O. Dekel, J. Keshat, S. Shalev-Shwartz, Y. Singer - JMLR (2006)
 
     """
-    def __init__(self, C=1.0, fit_intercept=True, n_iter=5, shuffle=True,
-                 verbose=0, loss="hinge", n_jobs=1, random_state=None,
-                 warm_start=False, class_weight=None):
+
+    def __init__(self, C=1.0, fit_intercept=True, max_iter=5, tol=1e-4,
```

Comment:
On 09/08/2015 02:33 AM, Olivier Grisel wrote:
>
> Now that we have a good stopping criterion, I think we should set 
> |max_iter=100| by default and expect the stopping criterion to kick in 
> before that in 99% of the cases.
>
Did someone do experiments on how well that works in practice?